server/src/ClientToDBAPI/InvestigationInfo/mainRoute.ts

```diff
@@ -14,8 +14,8 @@ investigationInfo.get('/staticInfo', (request: Request, response: Response) => {
 })
 
 investigationInfo.post('/updateInvestigationStatus', (request: Request, response: Response) => {
-    const { epidemiologyNumber, investigationStatus, endTime } = request.body;
```

Comment:
It is a bug fix that I inserted yesterday... Maybe yo overwrite my code?  inserted last night.

Reply:
*you